pkg/platform/runtime/artifact/download.go

```diff
@@ -39,6 +42,21 @@ func NewDownloadsFromBuild(buildStatus *headchef_models.V1BuildStatusResponse) (
 	return downloads, nil
 }
 
+func NewDownloadsFromBuildPlan(build bpModel.Build, artifacts map[strfmt.UUID]Artifact) ([]ArtifactDownload, error) {
+	var downloads []ArtifactDownload
+	for id := range artifacts {
+		for _, a := range build.Artifacts {
+			if a.Status == string(bpModel.ArtifactSucceeded) && a.TargetID == id && a.URL != "" {
+				if strings.Contains(a.MimeType, "application/x.artifact") || strings.Contains(a.MimeType, "application/x-activestate-artifacts") {
```

Comment:
Shouldn't these be equals, rather than contains?

Reply:
Whoops, yes you're right.